currency-conversion/src/main/java/lk/ac/ucsc/currencyconversion/ForexServiceProxy.java

```diff
@@ -0,0 +1,14 @@
+package lk.ac.ucsc.currencyconversion;
+
+import org.springframework.cloud.netflix.ribbon.RibbonClient;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient(name = "forex-service")
+@RibbonClient(name = "forex-service")
+public interface ForexServiceProxy {
+    @GetMapping("/currency-exchange/from/{from}/to/{to}")
+    public CurrencyConversionBean retrieveExchangeValue
+            (@PathVariable("from") String from, @PathVariable("to") String to);
```

Comment:
@Binurim PathVariable("from")  describe variable name in path only if its different from assigned variable name

Reply:
done